chef/cookbooks/cpe_chrome/resources/cpe_chrome.rb

```diff
@@ -94,5 +96,31 @@ def manage_chrome_osx
         ],
       }
     end
+
+    unless node['cpe_chrome']['mp']['UseMasterPreferences'] == false
+      mprefs = node['cpe_chrome']['mp']['FileContents'].reject { |v| v.nil? }
```

Comment:
Assuming this is always an array, if you're just getting rid of `nil` [using compact](https://ruby-doc.org/core-2.2.0/Array.html#method-i-compact) is another way to do that.
```
mprefs = node['cpe_chrome']['mp']['FileContents'].compact
```